app.py

```diff
@@ -0,0 +1,103 @@
+from main.mains import Amity
+import argparse
+
+x = Amity()
+
+
+# Set up amity
+def initialise_amity():
+    x.read_file('main/input.txt')
+    x.offices
+    x.populate()
+    x.allocate_rooms()
+
+
+# Prints office allocations
+def get_offices():
+
+    for office in x.offices:
+        print 'These are the people in: %s' % (office.name)
+        for person in office.people:
+            print person.name
+        print "\n"
+
+
+# Prints livingspace allocations
+def get_livingspaces():
+
+    for livingspace in x.living_spaces:
+        print 'These are the people in: %s' % (livingspace.name)
+        for person in livingspace.people:
+            print person.name
+        print "\n"
+
+
+# prints individual office with its occupants
+def individual_office(name):
+
+    for office in x.offices:
+        if office.name == name:
+            for person in office.people:
+                print person.name
+
+
+# prints individual living space occupants
+def individual_living(name):
+
+    for livingspace in x.living_spaces:
+        if livingspace.name == name:
+            for person in livingspace.people:
+                print person.name
+
+
+# Gets unallocated employees
+def get_unallocated():
+    """Return a list of unallocated employees"""
+    print [m.name for m in x.unallocated]
+
+
+# prints unallocated fellows
+def get_unallocated_fellows():
+    print [r.name for r in x.unallocated_fellows]
+
+initialise_amity()
+
+"""List of command line arguments"""
+parser = argparse.ArgumentParser(
```

Comment:
**Code Quality: Nitpick**
Nice work here using `argparse`. Can you refactor this to make use of docopt (http://docopt.org/). This will make your code even more cleaner and give you access to a lot more functionality. I will also be able to better assess your understanding of this bit.